extensions/pika/package.json

```
@@ -0,0 +1,50 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "pika",
+  "title": "Pika",
+  "description": "Beautify screenshots quickly using Pika.style",
+  "icon": "pika-icon.png",
+  "author": "rishi",
+  "categories": [
+    "Design Tools",
+    "Web",
+    "Media"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "openFromURL",
+      "title": "Open URL in Pika",
+      "subtitle": "Pika.style",
```

Normally we use the extension title here
```suggestion
      "subtitle": "Pika",
```